src/Sylius/Behat/CountryNameToCodeConverterInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Sylius\Behat;
```

Comment:
It's not dependent on `Core`, so I would place it in `Addressing` **component**.